app/assets/javascripts/schedule_form.js

```diff
@@ -1,61 +1,19 @@
 $(function () {
-  addLabListener();
-  addActivityListener();
-  addObjectiveListener();
+  $("#add-lab").addMore();
```

Comment:
Now what's really fun is you can give each of these DOM elements a `data-add-more` attribute and even provide other meta data / configurations of the behavoir and then just have a generic selector `$("[data-add-more]")` and you built a jQuery plugin. Awesome refactor!!!